pkg/lightning/backend/local/local.go

```diff
@@ -165,6 +165,13 @@ type File struct {
 	sstIngester    sstIngester
 	finishedRanges syncedRanges
 
+	// sst seq lock
+	seqLock sync.Mutex
+	// seq number for incoming sst meta
+	nextSeq atomic.Int32
```

Comment:
seems unnecessary to make `nextSeq` atomic as it is only used inside `e.seqLock`

Reply:
yes, fixed.